cypress/integration/with-security/helpers/generate_data.js

```diff
@@ -84,6 +84,7 @@ describe('Generating BWC test data with security', () => {
       });
     // update default expression to use `.es(*)` instead of `.opensearch(*)` for bwc
     cy.get('[class="view-line"]').type('.es(*)');
+    cy.get('[data-test-subj="visualizeEditorRenderButton"]').click();
```

Comment:
In this PR https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2720 we recently introduced a change where the default expression for timeline changed from .es(*) to .opensearch(*) , so now we have one more step in the workflow where we need to click `update` button when we create timeline vis before we `save`.  So this line is to get the update button click through.

Reply:
sorry I missed this! thanks for the update!